src/cgame/rocket/rocketDataSource.h

```diff
@@ -246,7 +246,7 @@ class RocketDataSource : public Rml::Core::Element, public Rml::Controls::DataSo
 	void AddCancelbutton()
 	{
 		init = true;
-		Rml::Core::Factory::InstanceElementText( this, va( "<button onClick=\"hide %s\">Cancel</button>", GetOwnerDocument()->GetId().c_str() ) );
+		Rml::Core::Factory::InstanceElementText( this, va( "<button onClick=\"hide %s\"><span>Cancel</span></button>", GetOwnerDocument()->GetId().c_str() ) );
```

Comment:
The circlemenu cancel button is generated by a Lua script, so whatever this is, it's not the circlemenu button.

Reply:
Ok then, dropped the commit.